code/modules/projectiles/guns/revolvers.dm

```diff
@@ -648,32 +658,58 @@
 
 /obj/item/weapon/gun/revolver/cmb
 	name = "\improper CMB Spearhead autorevolver"
-	desc = "An automatic revolver chambered in .357. Commonly issued to Colonial Marshals. It has three select fire options, safe, single, and burst."
+	desc = "An automatic revolver chambered in .357. Commonly issued to Colonial Marshals. It has two select fire options, single and burst."
```

Comment:
```suggestion
	desc = "An automatic revolver chambered in .357, often loaded with hollowpoints to prevent hull damages. Commonly issued to Colonial Marshals. It has two select fire options, single and burst."
```

Reply:
done